modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/utils/client.es.js

```diff
@@ -146,7 +146,47 @@ export const deleteMessageBoardThreadQuery = gql`
 	}
 `;
 
-export const getTagsQuery = gql`
+export const getTags = (
+	orderBy,
+	page = 1,
+	pageSize = 30,
+	search = '',
+	siteKey
+) => {
+	if (orderBy === 'latest-created') {
+		return client
+			.query({
+				query: getTagsOrderByDateCreatedQuery,
+				variables: {
+					page,
+					pageSize,
+					search,
+					siteKey,
+				},
+			})
+			.then((result) => ({
+				...result,
+				data: result.data.keywords,
+			}));
+	}
+
+	return client
+		.query({
+			query: getTagsOrderByNumberOfUsagesQuery,
+			variables: {
+				page,
+				pageSize,
+				search,
+				siteKey,
+			},
+		})
+		.then((result) => ({
+			...result,
```

Comment:
I don't think this is needed (I know we are doing it in other query calls) but I think you just need the data and you can unwrap it with ({data: {keywords}}) or something like that.

But it's ok, it's a good practice to return everything.